proto/regen/ecocredit/basket/v1/events.proto

```diff
@@ -0,0 +1,85 @@
+syntax = "proto3";
+
+package regen.ecocredit.basket.v1;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/timestamp.proto";
+import "cosmos/base/v1beta1/coin.proto";
+import "regen/ecocredit/basket/v1/types.proto";
+
+option go_package = "github.com/regen-network/regen-ledger/x/ecocredit/basket/v1";
+
+// EventCreate is an event emitted when a basket is created.
+message EventCreate {
```

Comment:
I didn't include `id` because it is used internally for state. Also didn't include `name` because it we have the generated `basket_denom` following creation. Let me know if that makes sense.

Reply:
I like it - events should only contain the necessary data to identify object or transaction.

Reply:
makes sense